docs/nodes/maintain/avalanchego-config-flags.md

```diff
@@ -152,6 +152,24 @@ Specifies the directory that contains chain configs, as described [here](chain-c
 
 As an alternative to `--chain-config-dir`, chains custom configurations can be loaded altogether from command line via `--chain-config-content` flag. Content must be base64 encoded.
 
+#### `--chain-aliases-file` (string):
+
+Path to JSON file that defines aliases for Blockchain IDs. Defaults to `~/.avalanchego/configs/chains/aliases.json`. This flag is ignored if `--chain-aliases-file-content` is specified. Example content:
+
+```json
+{
+  "2K33xS9AyP9oCDiHYKVrHe7F54h2La5D8erpTChaAhdzeSu2RX": [
+    "Swimmer"
+  ]
+}
+```
+
+The above example aliases the Blockchain whose ID is `"2K33xS9AyP9oCDiHYKVrHe7F54h2La5D8erpTChaAhdzeSu2RX"` to `"Swimmer"`. Chain aliases are added after adding primary network aliases and before any changes to the aliases via the admin API. This means that the first alias included for a Blockchain on a Subnet will be treated as the `"Primary Alias"` instead of the full blockchainID. The Primary Alias is used in all metrics and logs.
```

Comment:
Can you clarify `Chain aliases are added after adding primary network aliases and before any changes to the aliases via the admin API.`?

Reply:
@aaronbuchwald ^^